collection-scripts/targeted_crs

```diff
@@ -119,11 +119,14 @@ if [ ! -z "${vm_resources}" ]; then
 
         # Gather VMs PersistentVolumeClaims
         for pvc_name in $(echo $vm_data | jq -r '.spec.template.spec.volumes[] .persistentVolumeClaim.claimName'); do
-          log_filter_query="$log_filter_query|$pvc_name"
-          dump_resource "persistentvolumeclaim" $pvc_name $target_ns
-
-          # Store PVC in list to allow populator pod logs gathering
-          echo "${target_ns},${pvc_name}" >> /tmp/pvcs
+          # In order to gather the scrach PVCs in warm migrations
+          for pvc in $(/usr/bin/oc get pvc -n ${target_ns} --no-headers | grep $pvc_name | awk '{print $1}'); do
```

Comment:
how about getting the name from the metadaa instead of using `awk` (that relies on the order of the properties)

Reply:
doesn't work with the current change. need to see how to proper loop. don't merge.